tn_api_v2_core.yml

```diff
@@ -575,6 +578,10 @@ components:
                     type: string
                     description: Flight number. Example = "391"
                     example: "391"
+                plating_carrier:
```

Comment:
Not the plating carrier, it's the "operating carrier" that matters on this one @sunny-trip-ninja 

plating carrier = who sells it, takes the $. If for example Air Canada sells a United flight, this is Air Canada.
operating carrier = who actually operates the plane - United Airlines in this case.
marketing carrier = I'm actually a little less certain about this, but sometimes there's codeshare flight numbers so for example AC661 and UA1234 may both be on the same plane operated by United Airlines but in this case the marketing carrier could be either UA or AC.

Reply:
good to know will update!